packages/headless/src/api/search/search/stemming.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Terms to highlight (with stemming) in the results
```

Comment:
@jpmarceau : What do you think we should add as a description for "stemming" ?

Benjamin's point is right I think, but stemming can be something complicated to explain.

Maybe this article ? https://docs.coveo.com/en/1576/searching-with-coveo/about-stemming

Reply:
I like Benjamin's idea of an example. It should answer most people's questions. I'd just tweak it a little because I think it'd be clearer if we distinguished stemming from query corrections.
I also like the idea of linking to that document for people who'd like to know more, since explaining what stemming is takes quite a bit of space. I'm not sure I'd do that in code annotations.

```suggestion
 * Terms to highlight (with stemming) in the results.
 * E.g., for the query "indexing consultant", where a result contains "Techzample is a consulting firm specializing in search indexes. It employs consultants that can analyze and improve your search index.", the `TermsToHighlight` might be `{ "indexing": ["indexes", "index"], "consultant": ["consulting", "consultants"] }`.
 * To learn more about stemming, see [About Stemming](https://docs.coveo.com/en/1576).
```